lib/assesments/assessment_2/fi-as2-hira/Screens/My prpofile/widgets/fi_as2_hira_part3.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+
+class H1Part3 extends StatelessWidget {
+  const H1Part3({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    Widget smallspace = const SizedBox(
+      height: 16,
+    );
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+      children: [
+        Container(
+          height: 275,
+          width: 195,
+          decoration: BoxDecoration(
+            borderRadius: BorderRadius.circular(12),
+          ),
+          child: Image.asset('assets/fi-as2-hira-assets/pic1.png'),
+        ),
+        Column(
+          children: [
+            Container(
```

Comment:
Could have used a `List<String> images` where all the paths are given and you can use `map` to iterate the list here